### DISCUSSION TITLE
training data: good enough quality to train AI

### DISCUSSION THREAD

**Discussion by schneich** [jomjol/AI-on-the-edge-device#2602] [General] [answered]
Hi,

I am still facing some misreadings. Especially "3" is misinterpreted a lot. (https://github.com/jomjol/AI-on-the-edge-device/discussions/2423 | https://github.com/jomjol/AI-on-the-edge-device/discussions/1949#discussioncomment-6958846) Before I seriously start gathering the training data, I would like to know, if this is good enough quality for you to train the AI.

Sample data from the last 8 hours: 
[labeled.zip](https://github.com/jomjol/AI-on-the-edge-device/files/12567780/labeled.zip)

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/84641760/4e4aef54-bf34-4df4-bf76-4a725b12d63c)

Thanks for feedback,
Chris

**Comment by caco3** [comment]
@jomjol

**Comment by jomjol** [comment]
Hi Chris,
sorry for the delayed answer. The contrast is not very good. But I would give it a try.

Let me know, when you have a full set of training data.

**Comment by schneich** [comment, marked as answer]
Hi @jomjol,

I started to collect the data, but I was very unhappy with the reflections. My water meter has a glas, which is ~5mm above the numbers. _Where there is light, there will be reflections_. 

Yesterday, I spend most of the day trying different arrangements of my light strip. I guess I found an acceptable compromise: I was able to reduce the reflections to a bare minimum and the contrast did not get much worse. Anyway, the AI seems to better recognize the difficult numbers.

I will keep an eye on it for the next couple of days. If there are some miss readings, I will provide a training set of data.

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/84641760/1618163d-d78a-44f4-94b5-cb83a474424c)


best regards from Switzerland,
Chris

**Reply by schneich** [reply]
Hi @jomjol,

just wanted to let you know, that the rearrangement of my light strip did it. So far, no misreadings anymore.

I guess I will close this issue for now. Thank you for your feedback and all the effort on my questions :-). Very much appreciated.
Chris
